app/scripts/modules/core/loadBalancer/filter/LoadBalancerFilterCtrl.js

```diff
@@ -49,8 +50,45 @@ module.exports = angular.module('spinnaker.core.loadBalancer.filter.controller',
       loadBalancerFilterService.updateLoadBalancerGroups(app);
     }
 
+    this.getAvailabilityZoneHeadings = () => {
+      let selectedRegions = LoadBalancerFilterModel.getSelectedRegions();
+      let availableRegions = this.getRegionHeadings();
+
+      return selectedRegions.length === 0 ?
```

Comment:
You could get away with `return selectedRegions.length ?` here, but I don't think we've established a convention around truthiness, and this is a little more explicit, so it's good.